lib/clients/aws-client/lambda-client.js

```diff
@@ -147,7 +147,7 @@ module.exports = class LambdaClient extends AbstractAwsClient {
      */
     _generateFunctionName(skillName, profile, alexaRegion) {
         const validSkillName = stringUtils.filterNonAlphanumeric(skillName.toLowerCase()).substring(0, 22);
-        const validProfile = stringUtils.filterNonAlphanumeric(profile.toLowerCase());
+        const validProfile = stringUtils.filterNonAlphanumeric(profile.toLowerCase()).substring(0, 15);
```

Comment:
this change is a little bit outside of the scope. Deploy fails if profile name is too long. It happens in the situation when using __ENVIRONMENT_ASK_PROFILE__ profile.